.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Instructions For Filing a Bug: https://github.com/GoogleWebComponents/model-viewer/blob/master/CONTRIBUTING.md#filing-bugs -->
+### Description
+
+Please provide a detailed description of the bug, how to reproduce it, and the
+expected behavior.  Always include a code snippet, screenshot, any errors
+reported in the console, and the model to help us understand and fix the
+problem.
+
+#### Live Demo
+<!-- glitch.me starting point (remix and edit -- must be logged in to persist!) -->
+https://glitch.com/edit/#!/model-viewer
+<!-- ...or provide your own repro URL -->
+
+### Browsers Affected
+<!-- Check all that apply and please include the version tested -->
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
+- [ ] Safari 11
+- [ ] Safari 10
+- [ ] IE 11
```

Comment:
I would recommend that we don't suggest support for Safari < 11. The Polymer team rule is that we support back to ($LATEST_VERSION - 1) of all major browsers, and IE11.

Firefox 62 is an interesting case because it is the last version of Firefox without Web Components.

We also care about iOS / Android browsers as special cases due to AR features. Also Magic Leap's Helios.

Reply:
Let me drop the versions, and I'll add Helios and an OS picker.